pkg/controllermanager/controller/shoot/shoot_control_reconcile.go

```diff
@@ -259,6 +259,11 @@ func (c *defaultControl) reconcileShoot(o *operation.Operation, operationType ga
 			Fn:           flow.SimpleTaskFn(botanist.DeploySeedLogging).RetryUntilTimeout(defaultInterval, defaultTimeout),
 			Dependencies: flow.NewTaskIDs(waitUntilKubeAPIServerIsReady, initializeShootClients, waitUntilVPNConnectionExists, waitUntilWorkerReady),
 		})
+		_ = g.Add(flow.Task{
```

Comment:
Remove this task - take a look how it's done here: https://github.com/gardener/gardener/pull/1000/files#diff-a68e0b6e59734836dbbde01299666e87R895